.github/workflows/authenticate_test.yml

```diff
@@ -13,22 +13,29 @@ on:
 env:
   CARGO_TERM_COLOR: always
   RUST_BACKTRACE: full
+  DOCKER_COMPOSE_VERSION: 2.32.4
 
 jobs:
-  # PasswordAuthenticator
   build:
-    timeout-minutes: 60
+    name: Run Authentication tests
     runs-on: ubuntu-latest
-    services:
-      scylladb:
-        image: scylladb/scylla
-        command: --authenticator PasswordAuthenticator
-        ports:
-          - 9042:9042
-        options: --health-cmd "cqlsh --username cassandra --password cassandra --debug" --health-interval 5s --health-retries 30
     steps:
-    - uses: actions/checkout@v3
+    - name: Checkout
+      uses: actions/checkout@v3
+
     - name: Update rust toolchain
       run: rustup update
+
+    - name: Start the cluster
+      run: docker compose -f test/cluster/docker-compose-passauth.yml up -d
+
     - name: Run tests
       run: RUST_LOG=trace cargo test --verbose authenticate_superuser -- custom_authentication --ignored
```

Comment:
:wrench: Let's pass `SCYLLA_URI=172.43.0.2` directly here, to the `cargo test` invocation.